### PR TITLE
updating kube api version

### PR DESCRIPTION
**Pull request by kalexand-rh**
@wmengRH, when you're back, will you PTAL at these kube version updates?

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@wmengRH bump - can you please review?

File: modules/dr-recover-lost-control-plane-hosts.adoc [hunk 1]
```diff
@@ -85,7 +85,7 @@ In a terminal that has access to the cluster, run the following command to verif
 ----
 $ oc get nodes -l node-role.kubernetes.io/master
 NAME                                         STATUS   ROLES    AGE   VERSION
-ip-10-0-143-125.us-east-2.compute.internal   Ready    master   46m   v1.13.4+db7b699c3
+ip-10-0-143-125.us-east-2.compute.internal   Ready    master   46m   v1.14.0+f667219f4
```
Comment (liangxia) [inline modules/dr-recover-lost-control-plane-hosts.adoc]: I got,
```
$ oc get nodes 
NAME                                         STATUS   ROLES    AGE     VERSION
ip-10-0-129-61.us-east-2.compute.internal    Ready    worker   176m    v1.14.6+c4799753c
ip-10-0-130-169.us-east-2.compute.internal   Ready    master   4h39m   v1.14.6+c4799753c
ip-10-0-134-195.us-east-2.compute.internal   Ready    worker   4h33m   v1.14.6+c4799753c
ip-10-0-140-215.us-east-2.compute.internal   Ready    worker   176m    v1.14.6+c4799753c
ip-10-0-153-190.us-east-2.compute.internal   Ready    worker   4h33m   v1.14.6+c4799753c
ip-10-0-153-201.us-east-2.compute.internal   Ready    master   4h39m   v1.14.6+c4799753c
ip-10-0-167-172.us-east-2.compute.internal   Ready    master   4h39m   v1.14.6+c4799753c
```

Comment (kalexand-rh) [reply modules/dr-recover-lost-control-plane-hosts.adoc]: Thank you @liangxia! I'll update them all to `v1.14.6+c4799753c` and merge.

**Review by bobfuru — COMMENTED** [review]
Lines 6-20 use lowercase "pod" but I believe we should be capitalizing as "Pod": https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting
Other than that and one minor edit, LGTM.

File: modules/nodes-nodes-viewing-listing.adoc [hunk 2]
```diff
@@ -13,9 +13,9 @@ You can get detailed information on the nodes in the cluster.
...
 NAME                   STATUS    ROLES     AGE       VERSION
-master.example.com     Ready     master    7h        v1.13.4+b626c2fe1
-node1.example.com      Ready     worker    7h        v1.13.4+b626c2fe1
-node2.example.com      Ready     worker    7h        v1.13.4+b626c2fe1
+master.example.com     Ready     master    7h        v1.14.6+c4799753c
+node1.example.com      Ready     worker    7h        v1.14.6+c4799753c
+node2.example.com      Ready     worker    7h        v1.14.6+c4799753c
 ----
 
 * The `-wide` option provides provides additional information on all nodes.
```
[the remark this answers is not in the document]
Comment (kalexand-rh) [reply modules/nodes-nodes-viewing-listing.adoc]: This is out of scope of the change I was making, but I'll open a new PR to fix it. (I'm not touching the pod capitalization issues - I can't tell from your comment which module they're in.)

[the remark this answers is not in the document]
Comment (bobfuru) [reply modules/nodes-nodes-viewing-listing.adoc]: No worries...I have been capitalizing in Storage docs (e.g., https://docs.openshift.com/container-platform/4.1/storage/understanding-persistent-storage.html).

**Pull request merged**